src/main/java/org/lisoft/lsml/model/item/AmmoWeapon.java

```diff
@@ -81,19 +89,48 @@ public Ammunition getAmmoType() {
         return ammoType;
     }
 
+    public int getVolleySize() {
+        return volleysize;
+    }
+    
+    public int getAmmoPerShot() {
+        return ammoPerShot;
+    }
+    
     public int getBuiltInRounds() {
-        return hasBuiltInAmmo() ? getAmmoPerPerShot() : 0;
+        return hasBuiltInAmmo() ? super.getRoundsPerShot(): 0;
     }
 
     @Override
     public double getCoolDown(Collection<Modifier> aModifiers) {
         return isOneShot() ? Double.POSITIVE_INFINITY : super.getCoolDown(aModifiers);
     }
 
+     /**
+     * The unmodified time between shots. C.f. {@link #getExpectedFiringPeriod(Collection)}.
+     * <p>
+     * Note that this is different from cooldown which is the time the weapon is unavailable between uses, this is
+     * the time between activations of the weapon. In particular this includes the time that it takes to charge
+     * a gauss rifle, the burn time of lasers, the volley delay from LRMs etc that is not included in cooldown.
+     *
+     * @return The firing period [seconds]
+     * @param aModifiers The modifiers to apply from quirks etc.
+     * there can also be additional delay for hardpoint volley size limitations. not accounted for, yet.
```

Comment:
Sestrand said:  I moved this from the base to here because it only applies to the
ammoweapon weapons.  so nothing to inherit.  but I can move the free text
prior.

Reply:
The comment in Weapon.java hasn't been moved. There were a few lines added to it. 

What I'm asking for is something like this:

```
/**
 * {@inheritDoc}
 *
 * Also accounts for volley size of weapons that have such limitations.
 * 
 * @param aModifiers {@inheritDoc}
 * @return {@inheritDoc}
 * /
```

And then remove the mention of volley size in the base method javadoc because it doesn't actually do that. Or alternatively to keep the mention in the [base method](https://github.com/lisongmechlab/lsml/pull/778/files/7a9c2b454ddca52f15168c84ec938ef1ffad4c1b#diff-f210accc0fc5382ea421d7526373d66e1252c07ce4aed7ba951ce372884e73a1) and just use /** `{@inheritDoc}` */ here. Or leave the comment empty (the IDE would figure out the inheritance).

Reply:
ok, I changed some stuff... probably still not what you wanted, but I am not that familiar with how the Docs work.

Reply:
Most IDE's will render the JavaDoc format to HTML tooltips with links and inlining of inheritDoc etc. This looks correct to me.